### PR TITLE
improve error message when import failed

### PR DESCRIPTION
**Pull request by kmaehashi**
Import error message imporovements from https://github.com/cupy/cupy/pull/954

### CODE REVIEW

File: cupy/__init__.py [hunk 1]
```diff
@@ -14,10 +14,20 @@
...
-    msg = ('CuPy is not correctly installed. Please check your environment, '
-           'uninstall CuPy and reinstall it with `pip install cupy '
-           '--no-cache-dir -vvvv`.\n\n'
-           'original error: {}'.format(exc_info[1]))
+    msg = ('''\
+CuPy is not correctly installed.
+
+If you are using wheel distribution (cupy-cudaXX), make sure that the version of CuPy you installed matches with the version of CUDA on your host.
+Also, confirm that only one CuPy package is installed:
+  $ pip freeze | grep cupy
```
Comment (niboshi) [inline cupy/__init__.py]: Isn't it confusing for Windows users?

Comment (kmaehashi) [reply cupy/__init__.py]: OK, I removed the grep command.

**Comment by niboshi** [comment]
I wonder if we should include command line in an exception message, because the command prompt `$` is OS-dependent.

**Comment by niboshi** [comment]
Hmm, maybe I'm worrying too much.

**Comment by niboshi** [comment]
Jenkins, test this please

**Comment by niboshi** [comment]
LGTM!

**Pull request merged**